CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 ## Unreleased
 
+### New features
+
+You can now use relative links with your site.
+
+To your `config.rb` file, add the following lines:
+
+```
+set :relative_links, true
+activate :relative_assets
+```
+
+This may help with deploying to GitHub pages.
+
+- [#291: Support sites deployed on paths other than "/" (by generating relative links)](https://github.com/alphagov/tech-docs-gem/pull/291)
+
+Thanks [@eddgrant](https://github.com/eddgrant) for contributing this feature and the associated fixes!
+
```

Comment:
## 3.2.0

### New features 

You can now [configure your Tech Docs Template (TDT) to build your documentation site to use relative links to pages and assets](https://tdt-documentation.london.cloudapps.digital/configure_project/global_configuration/#build-your-site-using-relative-links).

Thanks [@eddgrant](https://github.com/eddgrant) for contributing this feature and the associated fixes.

This change was introduced in [pull request #291: Support sites deployed on paths other than "/" (by generating relative links)](https://github.com/alphagov/tech-docs-gem/pull/291).

Reply:
Yep!

Reply:
awesome - do you need me to do anything or are you ok to commit it in? I can review

Reply:
I'll commit it in

Reply:
Done!